robot/game.py

```diff
@@ -1,10 +1,37 @@
+import logging
+import time
 from enum import Enum
+from threading import Thread
 from typing import NewType
 
+import _thread
 from robot.board import Board
 
 Zone = NewType('Zone', int)
 
+LOGGER = logging.getLogger(__name__)
+
+
+def kill_after_delay(timeout_seconds, exit_message):
+    """
+    Interrupts main process after the given delay.
+    """
+
+    end = time.time() + timeout_seconds
+
+    def worker():
+        while time.time() < end:
+            time.sleep(0.1)
+
+        LOGGER.info("Timeout %r expired: %s", timeout_seconds, exit_message)
```

Comment:
This currently would display as:
`Timeout 150 expired: Game Over!`

I think this could be a bit confusing to people reading through the logs as it doesn't say what the number means.

`Timeout 150s expired: Game Over!`